libsplinter/src/network/connection_manager/heartbeat_monitor.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 Cargill Incorporated
```

Comment:
Any reason not to call this "heartbeat" for a shorter module name (without an underscore)?